Cubical/Algebra/Group/EilenbergMacLane/Properties.agda

```diff
@@ -828,41 +828,38 @@ Iso.leftInv (inducedFun-EM-rawIso e n) = h n
                ; (merid a i) k → merid (p a k) i}
 
-Iso→EMIso : {G : AbGroup ℓ} {H : AbGroup ℓ'}
-  → AbGroupEquiv G H → ∀ n → Iso (EM G n) (EM H n)
-Iso.fun (Iso→EMIso is n) = inducedFun-EM (GroupEquiv→GroupHom is) n
-Iso.inv (Iso→EMIso is n) = inducedFun-EM (GroupEquiv→GroupHom (invGroupEquiv is)) n
-Iso.rightInv (Iso→EMIso is zero) = Iso.rightInv (inducedFun-EM-rawIso is zero)
-Iso.rightInv (Iso→EMIso is (suc zero)) =
-  Iso.rightInv (inducedFun-EM-rawIso is (suc zero))
-Iso.rightInv (Iso→EMIso is (suc (suc n))) =
-  Iso.rightInv (mapCompIso (inducedFun-EM-rawIso is (suc (suc n))))
-Iso.leftInv (Iso→EMIso is zero) =
-  Iso.leftInv (inducedFun-EM-rawIso is zero)
-Iso.leftInv (Iso→EMIso is (suc zero)) =
-  Iso.leftInv (inducedFun-EM-rawIso is (suc zero))
-Iso.leftInv (Iso→EMIso is (suc (suc n))) =
-  Iso.leftInv (mapCompIso (inducedFun-EM-rawIso is (suc (suc n))))
-
-Iso→EMIso∙ : {G : AbGroup ℓ} {H : AbGroup ℓ'}
-  → (e : AbGroupEquiv G H)
-  → ∀ n
-  → Iso.fun (Iso→EMIso e n) (EM∙ G n .snd) ≡ EM∙ H n .snd
-Iso→EMIso∙ e zero = IsGroupHom.pres1 (e .snd)
-Iso→EMIso∙ e (suc zero) = refl
-Iso→EMIso∙ e (suc (suc n)) = refl
-
-Iso→EMIso⁻∙ : {G : AbGroup ℓ} {H : AbGroup ℓ'}
-  → (e : AbGroupEquiv G H)
-  → ∀ n
-  → Iso.inv (Iso→EMIso e n) (EM∙ H n .snd) ≡ EM∙ G n .snd
-Iso→EMIso⁻∙ e zero = IsGroupHom.pres1 (invGroupEquiv e .snd)
-Iso→EMIso⁻∙ e (suc zero) = refl
-Iso→EMIso⁻∙ e (suc (suc n)) = refl
-
-Iso→EMIsoInv : {G : AbGroup ℓ} {H : AbGroup ℓ'}
-  → (e : AbGroupEquiv G H) → ∀ n
-  → Iso.inv (Iso→EMIso e n) ≡ Iso.fun (Iso→EMIso (invGroupEquiv e) n)
+module _ {G : AbGroup ℓ} {H : AbGroup ℓ'} where
+  Iso→EMIso : AbGroupEquiv G H → ∀ n → Iso (EM G n) (EM H n)
+  Iso.fun (Iso→EMIso is n) = inducedFun-EM (GroupEquiv→GroupHom is) n
+  Iso.inv (Iso→EMIso is n) = inducedFun-EM (GroupEquiv→GroupHom (invGroupEquiv is)) n
+  Iso.rightInv (Iso→EMIso is zero) = Iso.rightInv (inducedFun-EM-rawIso is zero)
+  Iso.rightInv (Iso→EMIso is (suc zero)) =
+    Iso.rightInv (inducedFun-EM-rawIso is (suc zero))
+  Iso.rightInv (Iso→EMIso is (suc (suc n))) =
+    Iso.rightInv (mapCompIso (inducedFun-EM-rawIso is (suc (suc n))))
+  Iso.leftInv (Iso→EMIso is zero) =
+    Iso.leftInv (inducedFun-EM-rawIso is zero)
+  Iso.leftInv (Iso→EMIso is (suc zero)) =
+    Iso.leftInv (inducedFun-EM-rawIso is (suc zero))
+  Iso.leftInv (Iso→EMIso is (suc (suc n))) =
+    Iso.leftInv (mapCompIso (inducedFun-EM-rawIso is (suc (suc n))))
+
+  Iso→EMIso∙ : (e : AbGroupEquiv G H)
+    → ∀ n
+    → Iso.fun (Iso→EMIso e n) (EM∙ G n .snd) ≡ EM∙ H n .snd
+  Iso→EMIso∙ e zero = IsGroupHom.pres1 (e .snd)
+  Iso→EMIso∙ e (suc zero) = refl
+  Iso→EMIso∙ e (suc (suc n)) = refl
+
+  Iso→EMIso⁻∙ : (e : AbGroupEquiv G H)
+    → ∀ n
+    → Iso.inv (Iso→EMIso e n) (EM∙ H n .snd) ≡ EM∙ G n .snd
+  Iso→EMIso⁻∙ e zero = IsGroupHom.pres1 (invGroupEquiv e .snd)
+  Iso→EMIso⁻∙ e (suc zero) = refl
+  Iso→EMIso⁻∙ e (suc (suc n)) = refl
+
+Iso→EMIsoInv : {G : AbGroup ℓ} {H : AbGroup ℓ'} (e : AbGroupEquiv G H)
```

Comment:
This can be indented as well? And why not put e in the module as well?

Reply:
This can't be indented, because because invGroupEquiv e is an equivalence from H to G, and not G to H. But I can put e in the module, that's for sure.

Reply:
Oops